source/examples/demo-stages/include/demo-stages/ColorizeStage.h

```diff
@@ -18,6 +18,8 @@
 #include <gloperate/rendering/Camera.h>
 #include <gloperate/rendering/ScreenAlignedQuad.h>
 
+#include <demo-stages/demo-stages_api.h>
+
 
 namespace gloperate
```

Comment:
The namespace ```gloperate``` is not correct here (the sources does not belong to the gloperate library).
No namespace suffices for examples.